json-endpoints/jetpack/class.jetpack-json-api-updates-status-endpoint.php

```diff
@@ -26,9 +26,28 @@ protected function result() {
 			}
 		}
 
-		$result['jp_version'] = JETPACK__VERSION;
+		$result['jp_version']   = JETPACK__VERSION;
+		$result['is_vcs']       = $this->is_vcs();
+		$result['can_manage']   = $this->can_manage();
 
 		return $result;
 
 	}
+
+	private function is_vcs() {
+		if ( ! function_exists( 'is_vcs_checkout' ) ) {
```

Comment:
I think you should to use `method_exists` if you are checking for `WP_Automatic_Updater::is_vcs_checkout`.
I would use the `include_once` directly as I think is going to do some sort of the same check.